src/app/features/swap-form/reducer.ts

```diff
@@ -66,25 +130,28 @@ export const slice = createSlice({
     ) => {
       state.cashflowCalculator.predictedApy = value;
       if (stringToBigFloat(state.cashflowCalculator.predictedApy) < 0) {
```

Comment:
since it is no longer possible to end up with negative values, is this still required?

Reply:
[957310c](https://github.com/Voltz-Protocol/voltz-ui/pull/571/commits/957310cc8a0b03d2680b71d8f8ec5e98803f7150)